docs/book/user-guide/component-guide/model-deployers/model-deployers.md

```diff
@@ -90,54 +90,7 @@ zenml model-deployer register seldon --flavor=seldon \
    model version. Every model server that the Model Deployer provisions externally to deploy a model is represented
    internally as a `Service` object that may be accessed for visibility and control over a single model deployment. This
    functionality can be consumed directly from ZenML pipeline steps, but it can also be used outside the pipeline to
-   deploy ad-hoc models. The following code is an example of using the Seldon Core Model Deployer to deploy a model
-   inside a ZenML pipeline step:
-
-   ```python
-   from zenml.materializers import UnmaterializedArtifact
-   from zenml.environment import Environment
-   from zenml.integrations.seldon.model_deployers import SeldonModelDeployer
-   from zenml.integrations.seldon.services.seldon_deployment import (
-     SeldonDeploymentConfig,
-     SeldonDeploymentService,
-   )
-   from zenml.steps import (
-     STEP_ENVIRONMENT_NAME,
-     StepContext,
-     step,
-   )
-
-   @step(enable_cache=True)
-   def seldon_model_deployer_step(
-     context: StepContext,
-     model: UnmaterializedArtifact,
-   ) -> SeldonDeploymentService:
-     model_deployer = SeldonModelDeployer.get_active_model_deployer()
-
-     # get pipeline name, step name, and run id
-     step_env = Environment()[STEP_ENVIRONMENT_NAME]
-
-     service_config=SeldonDeploymentConfig(
-         model_uri=model.uri,
-         model_name="my-model",
-         replicas=1,
-         implementation="TENSORFLOW_SERVER",
-         pipeline_name = step_env.pipeline_name,
-         run_name = step_env.run_name,
-         pipeline_step_name = step_env.step_name,
-     )
-
-     service = model_deployer.deploy_model(
-         service_config, replace=True, timeout=300
-     )
-
-     print(
-         f"Seldon deployment service started and reachable at:\n"
-         f"    {service.prediction_url}\n"
-     )
-
-     return service
-   ```
+   deploy ad-hoc models. See the [seldon_model_deployer_step](https://apidocs.zenml.io/0.40.2/integration_code_docs/integrations-seldon/#zenml.integrations.seldon.steps.seldon_deployer.seldon_model_deployer_step) for an example of using the Seldon Core Model Deployer to deploy a model inside a ZenML pipeline step.
```

Comment:
Some of these links contain an explicit version which isn't the latest, is that intended?

Reply:
Good catch, updated them to latest 👍